src/main/java/de/plushnikov/intellij/plugin/processor/field/AccessorsInfo.java

```diff
@@ -175,7 +175,7 @@ public String removePrefix(String fieldName) {
   private boolean canPrefixApply(String fieldName, String prefix) {
     final int prefixLength = prefix.length();
     return prefixLength == 0 || fieldName.startsWith(prefix) && fieldName.length() > prefixLength &&
-      (!Character.isLetter(prefix.charAt(prefix.length() - 1)) || Character.isUpperCase(fieldName.charAt(prefixLength)));
+      (!Character.isLetter(prefix.charAt(prefix.length() - 1)) || Character.isUpperCase(fieldName.charAt(prefixLength)) || Character.isDigit(fieldName.charAt(prefixLength)));
```

Comment:
Good catch!
But after looking at this beautiful code here: https://github.com/rzwitserloot/lombok/blob/7993aa20cd20cdc030f90ee9f33c1b74c023d7ca/src/core/lombok/core/handlers/HandlerUtil.java#L452
I'm asking myself, maybe we should instead change `Character.isUpperCase(fieldName.charAt(prefixLength)` to `!Character.isLowerCase(fieldName.charAt(prefixLength)` ? What do you think?

Reply:
I think you are right! I'll change it.
Less code is a better code =)